### ISSUE TITLE
xDai Network not usable when you Click Add to Metamask

### ISSUE THREAD

**Issue by layer2index** [ethereum-lists/chains#210]
The UX in metamask takes you to the wrong URL when you try to check your transactions in Metamask after adding xDai Chain with Chainlist:
 
<img width="743" alt="Screen Shot 2021-04-07 at 9 02 40 AM" src="https://user-images.githubusercontent.com/79803006/113889209-ff586500-9790-11eb-918d-b2c515b631f1.png">

I manually added the xDai chain, and was using it fine and then removed the network from metamask and used Chainlist and it wasn't functional to use Metamask with xDai Chain.

<img width="360" alt="Screen Shot 2021-04-07 at 9 00 33 AM" src="https://user-images.githubusercontent.com/79803006/113889242-067f7300-9791-11eb-92f7-823746d8605e.png">

This is a HUGE regression. This isn't a Nice to have. 
The sole purpose for even using Chainlist to add a network to Metmask is shown to be broken.

<img width="328" alt="Screen Shot 2021-04-07 at 9 03 07 AM" src="https://user-images.githubusercontent.com/79803006/113889264-1008db00-9791-11eb-8242-e05a2cec68ff.png">

**Comment by ligi** [comment]
The infoURL was never intended to be a explorer url. It was intended to be a url to give information/context about the chain. like https://ethereum.org/ is for ethereum.

Now after #59 is implemented explorers have a separate field that should be consumed by http://chainlist.org - I will open an upstream issue.
